SimonSchwendele/displaylink-rpm
# Update to latest release from Displaylink (#238)

* Update to latest release from Displaylink

* Displaylink 5.7.0
* EVDI 1.13.1

* Add Fedora 38
* Remove older OS where kernel is no longer supported by evdi
* Add a patch for EL 8 and 9



* Updating the hashes on the Displaylink driver download and the bundled evdi module archive.



---------
